src/styles/header.scss

```diff
@@ -0,0 +1,68 @@
+@import 'fonts';
+@import 'colors';
+
+.containerHeader {
+  display: flex;
+  width: auto;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  font-family: $roboto-medium;
+  font-weight: $roboto-weight-medium;
+
+  &__logo {
+    height: 40px;
+  }
+}
+
+.navigation {
+  height: 60px;
+
+  &__list {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+    height: 100%;
+    align-items: center;
+    color: $color-font;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__item:first-child {
+    color: $color-links-header;
+  }
+
+  &__item &__link {
```

Comment:
The selector '.navigation__item &__link' seems to be incorrect. If you are trying to target a link within a navigation item, the correct syntax would be '.navigation__item .navigation__link'.